### PR TITLE
Pivotal ID # 175266840: Manual Set AccNo Sequence

### PR DESCRIPTION
**Pull request by jhoanmanuelms**
Generate next sequence having in mind the already existing submissions

### CODE REVIEW

File: submission/persistence-common/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/service/SubmissionQueryService.kt [hunk 1]
```diff
@@ -5,13 +5,22 @@ import java.time.OffsetDateTime
 
 interface SubmissionQueryService {
     fun getParentAccPattern(accNo: String): String?
+
```
Comment (jhoanmanuelms) [inline submission/persistence-common/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/service/SubmissionQueryService.kt:8]: BTW, I know I commented in a previous PR about this change but I think I got misunderstood. I think it looks more readable with spaces for the functions and no space for the variables.

Maybe we could agree to attach to the official kotlin standards in the future: https://kotlinlang.org/docs/reference/coding-conventions.html#formatting

File: submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt [hunk 2]
```diff
@@ -58,6 +60,8 @@ class SubmissionService(
     @RabbitListener(queues = [SUBMISSION_REQUEST_QUEUE], concurrency = "1-1")
     fun processSubmission(request: SubmissionRequestMessage) {
         logger.info { "received process message for submission ${request.submission}" }
+        Thread.sleep(30_000L)
```
Comment (Juan-EBI) [inline submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt:63]: why are we adding this?

**Review by Juan-EBI — COMMENTED** [review]
please consider comment before merge

File: submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt (hunk 2 above)
Comment (jhoanmanuelms) [reply submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt:63]: It was for testing purposes. I'll remove it, nice catch.

**Review by Juan-EBI — APPROVED** [review]

**Pull request merged**